src/global.c

```diff
@@ -387,3 +370,48 @@ git_global_st *git__global_state(void)
 }
 
 #endif /* GIT_THREADS */
+
+#ifdef GIT_WIN32 
+BOOL git__dllmain(HINSTANCE hInstDll, DWORD fdwReason, LPVOID lpvReserved)
```

Comment:
All `git__dllmain` implementations should be static

Reply:
Thank you for your review!
You're absolutely right of course.